demos/motion_tracking/motion_tracking_network.py

```diff
@@ -0,0 +1,229 @@
+# Copyright (C) 2023 Intel Corporation
+# SPDX-License-Identifier: BSD-3-Clause
+# See: https://spdx.org/licenses/
+
+import typing as ty
+from multiprocessing import Pipe
+from lava.lib.dnf.connect.connect import connect
+from lava.magma.core.run_conditions import RunSteps
+from demos.motion_tracking.dvs_file_input.process import DVSFileInput, PyDVSFileInputPM
```

Comment:
Does `demos` have to be added to the PYTHONPATH somewhere?

Reply:
No, demos is top level in lava-dnf and lava-dnf is part of the path, so it finds it.